templates/launch.html

```diff
@@ -157,11 +145,12 @@ <h3>Business Survey Metadata</h3>
         document.getElementById("flush-btn").disabled = true;
 
         const schema_name = document.getElementById("schema_name").value
+        const survey_type = document.querySelector(`#schema_name option[value=${schema_name}]`).dataset.surveyType
```

Comment:
Can we be consequent here and rewrite the document so we use `querySelector` everywhere or use older element selection method in the whole template?

Reply:
In this instance `querySelector` was needed because of what we are trying to select. I don't personally have a problem using both as I think both are still current and valid ways of selecting elements and we are using them for different purposes in this document. But Im happy to change it and just use `querySelector`

Reply:
`querySelector` is more modern API compared to `getElementById` and is explicit, you use css id and class selectors. I would keep it updated and consistent.

Reply:
Yeah it is also more explicit too to be fair. I've changed it now but can remove the commit?

Reply:
I think old api was used, so launcher can run in old browsers. I think querySelector has fairly large support in most browsers, so should be fine.

> I've changed it now but can remove the commit?

No, it is perfectly fine to keep.

> I would keep it updated and consistent.

Agreed, but there is a lot that could be done to the JS here but not strictly within scope of this. Happy to keep as is if it hasn't regressed anything.

Reply:
Yeah, I genuinely don't think it makes a difference, `querySelector` is a bit newer but `getElementById` is still valid, current and used. It's not obsolete or deprecated, it's just a slightly different way of doing it. As for using one API or two I don't think that makes a difference either to me in this case. But I don't have strong enough feelings on it either way so I made the change

Reply:
I just followed the `JavaScript: The Definitive Guide, 7th Edition`, these are author's words about `getElementById`👍

Reply:
But @MebinAbraham's comment about old browsers support is valid, `querySelector` is less compatible than the old method.

Reply:
Oh ok, I had a bit of a google to check and couldn't find anything current saying not to use it. Doesn't matter now anyway

Reply:
Some old browsers lack compatibility (IE and Firefox up to 2007/2008):
<img width="891" alt="Screenshot 2021-12-07 at 09 13 07" src="https://user-images.githubusercontent.com/53475968/145000752-9b21bc2c-a5a5-4828-afa7-ef38d8a6ef41.png">

Reply:
Yeah that was one reason I found to go with `getElementById` instead but as this is a dev tool that shouldn't really matter